### PR TITLE
@adobe/react-spectrum needs to include '*.css' in its sideEffects

### PR DESCRIPTION
**Pull request by TheMcMurder**
Adding a missing side effect configuration.

## ✅ Pull Request Checklist:

- [X] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues). **- NA**
- [X] Added/updated unit tests and storybook for this change (for new code or code which already has tests). **- NA**
- [X] Filled out test instructions. **- NA**
- [X] Updated documentation (if it already exists for this component).**- NA**
- [X] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/) **- NA**

## 📝 Test Instructions:

Stolen from https://github.com/adobe/react-spectrum/pull/707

Build an app with a minimal webpack config:
```js
const path = require("path");

module.exports = {
	entry: "./index.js",
	output: {
		filename: "index.js",
		path: path.resolve(__dirname, "public"),
	},
	mode: "production",
	module: {
		rules: [
			{
				test: /\.m?js$/,
				exclude: /(node_modules|bower_components)/,
				use: {
					loader: "babel-loader",
					options: {
						presets: ["@babel/preset-react"],
					},
				},
			},
			{
				test: /\.css$/i,
				use: ["style-loader", "css-loader"],
				// this was previously necessary:
				// sideEffects: true
			},
		],
	},
};
```
## 🧢 Your Project:

Intended to be used in Workfront, in the future.

### CODE REVIEW

**Comment by devongovett** [comment]
This package doesn't have any CSS files in it, it only re-exports other packages, so I'm not quite sure why this would be necessary. Could you give us more info about the problem you were having?

**Comment by TheMcMurder** [comment]
Sorry for the lack of context. I realized that this may actually be a bug in how rollup handles `sideEffects` in `package.json`

The context: I'm using rollup and including `react-spectrum` but no matter the configuration I couldn't get the css to be included. Once I removed the `sideEffects` flag I was able to see the `css` was correctly bundled so I created this issue.

Additional work this morning shows me that even when `sideEffects` looks like this:
```js
"sideEffects": [
  "*.css"
]
``` 
there are problems with the rollup build. I'm going to make sure my diagnosis is correct, but in the end this is probably not needed here. Sorry for the false alert. This can be closed and if I need to recreate it after inspecting the behavior a bit more I will.

TL;DR
- I'm pretty sure this is actually a bug in rollup.
- Rollup handles `sideEffects` differently than webpack.
- Webpack works just fine in this situation

**Comment by snowystinger** [comment]
Thanks for the extra info and taking the time to check it out! Feel free to reopen if you determine it's something else here.

**Comment by TheMcMurder** [comment]
Just closing the loop, I'm pretty confident this isn't an issue with this repo and I believe it is with how rollup handles the `sideEffects` key.

I opened an [issue there](https://github.com/rollup/rollup/issues/4084) instead to discuss it more.

I'm not very familiar with the `sideEffects` flag so I hope to learn more about it.
